.github/workflows/e2e-suite.yml

```diff
@@ -44,34 +44,21 @@ jobs:
         run: |
           timestamp=$(date +'%Y%m%d%H%M')
           report_filename="${timestamp}_cli_test_report.xml"
-          if ! pytest tests/integration --disable-warnings --junitxml="${report_filename}"; then
```

Comment:
Always didn't like this solution to get this step to pass for the next step. I have previously tried `continue-on-error` flag but that didn't work however with conditional statement like below `if: always()` or `if: success() || failure()` we can guarantee the next step is executed